addons/main/GPS2/fn_gps.sqf

```diff
@@ -0,0 +1,163 @@
+        _markerAlive setMarkerAlphaLocal 0.7;
+        _markerDead setMarkerAlphaLocal 0;
+
+        // keinen Spielernamen beim eigenen Marker oder wenn es deaktiviert ist anhängen
+        if (!(GVAR(SHOW_PLAYERNAMES)) || _x isEqualTo player) then
+        {
+            _markerAlive setMarkerTextLocal "";
+            continue;
+        };
+
+        // Spieler in Fahrzeug
+        if (vehicle _x != _x) then
+        {
+            private _vehicle = getText (configFile >> "cfgVehicles" >> typeOf (vehicle _x) >> "displayName");
+
+            // Spezialfall Drohne
+            if ((vehicle _x) in allUnitsUAV) then
+            {
+                private _operator = (UAVControl vehicle _x) select 0;
+                // UAV Operator ja/nein
+                if (!isNull _operator) then
+                {
+                    _markerAlive setMarkerTextLocal format["%1 (%2)", _vehicle, name _operator];
+                }
+                else
+                {
+                    _markerAlive setMarkerTextLocal format["%1 (---)", _vehicle];
+                };
+            } 
+            else
+            {
+                _markerAlive setMarkerTextLocal format["%1 (%2)", _vehicle, _name];
+            };
+        }
+        // Spieler zu Fuß
+        else
+        {
+            _markerAlive setMarkerTextLocal _name;
+        };
+    } forEach _unitsAlive;
+
+    // update dead players
+    {
+        private _name = name _x;
+        private _id = getPlayerID _x;
+        private _markerAlive = format["OPT_GPS_ALIVE_%1", _id];
+        private _markerDead = format["OPT_GPS_DEAD_%1", _id];
+
+        // Zeit des letzten Updates speichern (für späteren Cleanup)
+        GVAR(HashMapTime) set [_id, time];
+
+        // Todesmarker nur einmalig initialisieren
+        if !(_id in GVAR(HashMapDead)) then
+        {
+            createMarkerLocal [_markerDead, (vehicle _x)];
+            _markerDead setMarkerAlphaLocal 0;
+            _markerDead setMarkerTypeLocal "loc_Hospital";  // brauchbare Todesmarker: loc_Hospital, KIA
+            _markerDead setMarkerSizeLocal [0.8, 0.8];
+            _markerDead setMarkerColor "ColorRed";
+            GVAR(HashMapDead) set [_id, _markerDead];
+        };
+
+        _markerDead setMarkerPosLocal (vehicle _x);
+        _markerDead setMarkerDirLocal getDirVisual (vehicle _x);
+        _markerDead setMarkerAlphaLocal 1;
+        _markerAlive setMarkerAlphaLocal 0;
+
+        // keinen Verwundeten-Namen beim eigenen Marker oder wenn es deaktiviert ist anhängen
+        if (!(GVAR(SHOW_PLAYERNAMES)) || _x isEqualTo player) then
+        {
+            _markerDead setMarkerTextLocal "";
+            continue;
+        };
+
+        _markerDead setMarkerTextLocal _name;
+    } forEach _unitsDead;
+}, 1 / GVAR(FPS), _this] call CFUNC(addPerFrameHandler);
+
+[{
```

Comment:
kann man das nicht in den anderen PFH einbauen ?

Reply:
Hm, ja eigentlich schon.